votes/0150-pr-issue-association.md

```diff
@@ -0,0 +1,51 @@
+# A Vote for PR Issue Association
+
+* Discussion at https://github.com/tikv/community/pull/150
+* Tracking issue https://github.com/tikv/community/issues/149
+
+## Proposal
+
+### Motivation
+
+To measure software evolution, we focus on features implemented, bugs fixed, and general enhancement. Issues document these items, while a pull request focuses on the change set and whether we implement it correctly.
+
+Normally, an issue documents a work item, and a pull request resolves it (despite cherry-picks). However, the relation between pull requests and issues is complicated in TiDB projects.
+
+* 1 issue with 0 PRs: question, bug vanished without a reason, etc. 
```

Comment:
How about adding more descriptions of these cases after this proposal, especially about `1 issue with n PRs` and `n issues with 1 PR`?

Reply:
Yes. Here is an example https://github.com/tikv/community/pull/150#discussion_r719277798 . I'm going to add more concrete cases.

Reply:
@Mini256 could you please on fulfilling examples here? A PR to this PR is acceptable.

Reply:
@Mini256 thanks for your suggestions! Integrated.

@rleungx you can read the updated proposal to see if there is still something required.